### DISCUSSION TITLE
I/O issue if direct LUN drived by VirtIO-SCSI with SCSI Pass-Through enabled

### DISCUSSION THREAD

**Discussion by CarlosCCastro** [oVirt/ovirt-site#3160] [Q&A]
Hi all,

I´m new on OVirt.
I´m tryng to present direct LUN as gatekeepers (king of raw disks to control comunication with the storage) - Storage Dell PowerMAX2000.

I´m facing that the device mapped to my VM has failed to open communication with storage:

My scenario:
VM: Oracle Linux Server release 7.9 - 5.4.17-2136.324.5.3.el7uek.x86_64
Direct LUN: SCSI Pass-Through enabled + allowed privileged I/O
OVirt: Software Version:4.4.8.6-1.el8

Anybody can help me with this?

Thank you!
Carlos Castro.

**Comment by sandrobonazzola** [comment]
oVirt 4.4 reached EOL long time ago. Please upgrade to 4.5.
